packages/tabs/src/useTabs.ts

```diff
@@ -9,58 +9,59 @@ import { useId } from '@zendeskgarden/container-utilities';
 import { useSelection } from '@zendeskgarden/container-selection';
 import { IUseTabsProps, IUseTabsReturnValue } from './types';
 
-export const useTabs = <Item>({
+export const useTabs = <Value>({
+  values,
   orientation = 'horizontal',
   idPrefix,
   ...options
-}: IUseTabsProps<Item> = {}): IUseTabsReturnValue<Item> => {
+}: IUseTabsProps<Value>): IUseTabsReturnValue<Value> => {
   const prefix = useId(idPrefix);
   const PANEL_ID = `${prefix}__panel`;
   const TAB_ID = `${prefix}__tab`;
-  const { selectedItem, focusedItem, getContainerProps, getItemProps } = useSelection<Item>({
+  const { selectedValue, focusedValue, getGroupProps, getElementProps } = useSelection<Value>({
+    values,
     direction: orientation,
-    defaultSelectedIndex: 0,
+    defaultSelectedValue: values[0],
     ...options
   });
 
-  const getTabListProps: IUseTabsReturnValue<Item>['getTabListProps'] = ({
+  const getTabListProps: IUseTabsReturnValue<Value>['getTabListProps'] = ({
     role = 'tablist',
     ...other
   } = {}) => ({
-    ...getContainerProps(other),
+    ...getGroupProps(other),
     role: role === null ? undefined : role,
     'aria-orientation': orientation,
     'data-garden-container-id': 'containers.tabs',
     'data-garden-container-version': PACKAGE_VERSION
   });
 
-  const getTabProps: IUseTabsReturnValue<Item>['getTabProps'] = ({
+  const getTabProps: IUseTabsReturnValue<Value>['getTabProps'] = ({
     role = 'tab',
-    index,
+    value,
```

Comment:
With the suggested `tabs` update, we'll be able to handle `aria-disabled` state here – which should be happening in the container rather than the component. Storybook will need a bit of love in order to fully demonstrate the tabs. See `useCombobox` `options` for details.